DataDog/dd-trace-py
# ci: test library installation (backport #2736)

* Test library install

Install and load the library on every supported platform
during the build process.

* Skip universal2 test on arm

* Test on Alpine Linux

* Fix YAML quirk

* Install Alpine build dependencies

* Use a dedicated step for smoke testing

* Check source package with twine

Co-authored-by: mergify[bot] <37929162+mergify[bot]@users.noreply.github.com>

## Description



## Checklist
- [ ] Added to the correct milestone.
- [ ] Library documentation is updated.
- [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).
